### ISSUE TITLE
create_gcov: support for gnu_debuglink

### ISSUE THREAD

**Issue by justanerd** [google/autofdo#63]
create_gcov doesn't seem to find the debug sections from seperate .debug files.
Is support  for this planned or can I achieve this without unstripped binaries ?

**Comment by danielcdh** [comment]
You just need to have the buildid in the separate .debug file.

**Comment by danielcdh** [comment]
I think this should have been fixed by https://github.com/google/autofdo/pull/65, pease re-open if otherwise.

**Comment by justanerd** [comment]
still seems to be an issue.
```
/bin/bash: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=ddb86174e0fcba237885c1a472d89f34133f7b8a, stripped
/usr/lib/debug/bin/bash.debug: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter *empty*, for GNU/Linux 3.2.0, BuildID[sha1]=ddb86174e0fcba237885c1a472d89f34133f7b8a, with debug_info, not stripped

create_gcov --binary=/bin/bash --profile=perf.data --gcov=sort.gcov -gcov_version=1
Log file created at: 2018/04/18 11:04:04
Running on machine: deepspace
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
W0418 11:04:04.292893 91105 addr2line.cc:41] File '/bin/bash' has no .debug_info section.  
W0418 11:04:04.292918 91105 addr2line.cc:41] File '/bin/bash' has no .debug_ranges section.  
W0418 11:04:04.292922 91105 addr2line.cc:41] File '/bin/bash' has no .debug_line section.  
W0418 11:04:04.292979 91105 module_grouper.cc:60] Cannot read compilation info from binary. Please use -frecord-compilation-info-in-elf when building the binary
```

**Comment by ravipal** [comment]
> I think this should have been fixed by #65, pease re-open if otherwise.

This issue is still not fixed. Please re-open

**Comment by danilaml** [comment]
Still an issue on latest master.
